frontend/src/App.tsx

```diff
@@ -21,16 +23,102 @@ const App = () => {
-              <Route exact path="/employeeAccount/new" render={() => <CreateEmployee />} />
+              <Route
+                exact
+                path="/project"
+                render={() => (
+                  <React.Fragment>
+                    <Sidebar />
+                    <CreateProject edit="false" />
+                  </React.Fragment>
+                )}
+              />
+              <Route
+                exact
+                path="/project/:id"
+                render={({ match }) => (
+                  <React.Fragment>
+                    <Sidebar />
+                    <EditProject id={match.params.id} />
+                  </React.Fragment>
+                )}
+              />
+              <Route
+                exact
+                path="/projects"
+                render={() => (
+                  <React.Fragment>
+                    <Sidebar />
+                    <ViewProject />
+                  </React.Fragment>
+                )}
+              />
+              <Route
+                exact
+                path="/employees"
+                render={() => (
+                  <React.Fragment>
+                    <Sidebar />
+                    <LogHours />
+                  </React.Fragment>
+                )}
+              />
+              <Route
+                exact
+                path="/error"
+                render={() => (
+                  <React.Fragment>
+                    <Sidebar />
+                    <PageNotFound />
+                  </React.Fragment>
+                )}
+              />
+              <Route
+                exact
+                path="/businessAccount/new"
+                render={() => (
+                  <React.Fragment>
+                    <div style={{ paddingTop: '75px' }}>
+                      <Sidebar />
+                      <CreateBusinessAccount />
+                    </div>
+                  </React.Fragment>
+                )}
+              />
+              <Route
+                exact
+                path="/clientAccount/new"
+                render={() => (
+                  <React.Fragment>
+                    <div style={{ paddingTop: '75px' }}>
+                      <Sidebar />
+                      <CreateClientAccount />
+                    </div>
+                  </React.Fragment>
+                )}
+              />
+              <Route
+                exact
+                path="/employeeAccount/new"
+                render={() => (
+                  <React.Fragment>
+                    <div style={{ paddingTop: '75px' }}>
+                      <Sidebar />
+                      <CreateEmployee />
+                    </div>
+                  </React.Fragment>
+                )}
+              />
               <Route exact path="/login" render={() => <Login />} />
-              <Route exact path="/" render={() => <div>root</div>} />
+              <Route
```

Comment:
I don't think the root should have the side bar.

Reply:
It has been removed!